plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHABase.java

```diff
@@ -32,9 +32,10 @@
     private static final Logger s_logger = Logger.getLogger(KVMHABase.class);
     private long _timeout = 60000; /* 1 minutes */
     protected static String s_heartBeatPath;
+    protected static String s_heartBeatPathRbd;
     protected long _heartBeatUpdateTimeout = 60000;
     protected long _heartBeatUpdateFreq = 60000;
-    protected long _heartBeatUpdateMaxTries = 5;
+    protected long _heartBeatUpdateMaxTries = 10;
```

Comment:
Instead of changing the existing value, is it better to use new variable for ceph?